NavigationReactNative/src/TabBar.tsx

```diff
@@ -54,9 +55,14 @@ class TabBar extends React.Component<any, any> {
         var TabBar = primary ? NVTabBar : TabBarPager;
         var TabView = primary ? NVTabNavigation : (!I18nManager.isRTL ? NVTabLayout : NVTabLayoutRTL);
         TabView = Platform.OS === 'android' ? TabView : NVSegmentedTab;
+
+        var constants = Platform.OS === 'android' ? (UIManager as any).getViewManagerConfig('NVTabNavigation').Constants : null;
+        labelVisibilityMode = !(labelVisibilityMode === 'selected' && tabBarItems.length > 3) ? labelVisibilityMode : 'auto';
         var tabLayout = (Platform.OS === 'android' || !primary) && (
             <TabView
                 bottomTabs={bottomTabs}
+                labelVisibilityMode={Platform.OS === 'android' ? constants.LabelVisibility[labelVisibilityMode] : null}
```

Comment:
Ok, you don't need another platform check then. You can use optional chaining instead
```jsx
labelVisibilityMode={constants?.LabelVisibility[labelVisibilityMode]}
```

Reply:
Good spot, I'll add this in